f/fcs/fn_fcsInit.sqf

```diff
@@ -0,0 +1,118 @@
+// F3 - FCS/Commander's Override system
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+// Originally from: https://github.com/NikkoJT/njt_fcs (integrated with permission)
+
+/*
+	This component adds three main functions to a vehicle it's called on:
+	1. Commander's Override
+		The Commander's Override allows the commander to automatically point the gun at whatever object they're looking at.
+	2. FCS failure simulation
+		When the vehicle is hit by a high-calibre weapon, there is a small chance of an FCS failure, disabling the TC override, laser rangefinder, and NVDs, and force-setting the gunner's zeroing.
+		A unit with the repair trait can reset the FCS from the gunner's seat.
+		(If and when BI adds the ability to disable turret stabilisation by script, it will do that too)
```

Comment:
Is this different from `enableDirectionStabilization`?

Reply:
Yes, `enableDirectionStabilization` only affects turrets with automatic lock-on stabilization (e.g. UAV cameras and targeting pods). At the moment there's no way to control tank-type stabilisation through sqf.